packages/components/src/tooltip/index.tsx

```diff
@@ -73,7 +73,6 @@ function Tooltip( props: TooltipProps ) {
 	return (
 		<>
 			<Ariakit.TooltipAnchor
-				onBlur={ tooltipStore.hide }
```

Comment:
Could you please elaborate on this change?

Reply:
Yes, thank you for asking! I meant to leave a comment here when I opened the PR.

There was a conflict between the `onBlur` added to the Tooltip anchor and the function `useCopyToClipboard` in the `ColorPicker` component: https://github.com/WordPress/gutenberg/blob/70f787b79468c938ce2d0335ffa768e59e455cc3/packages/components/src/color-picker/color-copy-button.tsx#L21

In this function, the `focus` is reset, resulting in the tooltip hiding on copy button click before displaying the 'copied' text:

https://github.com/WordPress/gutenberg/blob/23bb9301902bb5825480240bd5aaec9d3cc3c380/packages/compose/src/hooks/use-copy-to-clipboard/index.js#L52-L59

Originally, we added the `onBlur` to the tooltip to replicate the legacy behavior where the tooltip is hidden when leaving the document. This was discovered through failing tests; but now, the tests pass without it. The [utility function](https://github.com/WordPress/gutenberg/pull/48440/files#diff-d040fe8f8aee3cd48e8f849d4fb7f848570f7f27fac6c85557a56538aa84a566) was added after, so it's likely the failure was related to leaky tests instead. 

I discussed removing this or adding it as a prop with @ciampo, and we decided to remove it for now. And if we find a use-case where it makes sense to have it, we could add it as an optional prop.

Reply:
Makes sense to me, thanks 👍

Reply:
We may need to consider adding a prop to customize this behaviour, since this change is related to a regression #57204